_software/2020-02-27-MNPRX.md

```diff
@@ -72,6 +72,23 @@ Combine MNPRX with Arnold or any other offline renderer for unmatched flexibilit
 <!-- Pricing section -->
 {% include mnprx-pricing.html %}
 
+<div>
+  <h2>Requirements</h2>
+<div class="aio-requirements" markdown="1">
+  <div class="aio-specs">    
+    <img width="40" src="/images/maya.svg" alt="Maya"/>
```

Comment:
add svgs (maya and gpu) in MNPRX folder as to not crowd the root images directory